### ISSUE TITLE
Adjust 韃 (U+97C3, CN only) and 韆 (U+97C6, CN only) to improve balance

### ISSUE THREAD

**Issue by CoolMarvel43** [adobe-fonts/source-han-serif#178]
The 辶 radical looks weird and off-balanced. Please adjust to match TW/HK/JP/KR proportions, which are fine.

<img width="642" alt="Screenshot 2023-03-03 at 14 26 29" src="https://user-images.githubusercontent.com/10541683/222647551-f293b3f1-0acf-4981-8781-c6a95e9e2d16.png">

**Comment by Marcus98T** [comment]
Not fixed in Noto Serif v2.002. This issue will remain open.

<img width="235" alt="Screenshot 2023-08-17 at 20 02 12" src="https://github.com/adobe-fonts/source-han-serif/assets/10541683/9f37b6d7-e4aa-4050-b7e7-be1eee53bdce">

**Comment by Wenti-D** [comment]
Well, in Unicode Standard, these two glyphs are just like current in SHS.
![image](https://github.com/adobe-fonts/source-han-serif/assets/73067578/a2e341b2-16d3-4c27-9437-d5626143e376)
![image](https://github.com/adobe-fonts/source-han-serif/assets/73067578/03a9f91c-f817-48c3-8b94-2ec773c025c4)

**Comment by Marcus98T** [comment]
> Well, in Unicode Standard, these two glyphs are just like current in SHS. ![image](https://private-user-images.githubusercontent.com/73067578/295122463-a2e341b2-16d3-4c27-9437-d5626143e376.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ3ODk5OTMsIm5iZiI6MTcwNDc4OTY5MywicGF0aCI6Ii83MzA2NzU3OC8yOTUxMjI0NjMtYTJlMzQxYjItMTZkMy00YzI3LTk0MzctZDU2MjYxNDNlMzc2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA5VDA4NDEzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ5Mzg3MTk0NjQ5ZDgyYmMxOGU2ZjFhNGNjMzI5NzkxZGVmMjRiYzllMmNiNTQyYzM2YzJkYjk3NjM0OThhZTUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-H_jOBTY-v_b4rxM0DGaAhlqK4n9I3eD4E5JLfU_jbw) ![image](https://private-user-images.githubusercontent.com/73067578/295122492-03a9f91c-f817-48c3-8b94-2ec773c025c4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ3ODk5OTMsIm5iZiI6MTcwNDc4OTY5MywicGF0aCI6Ii83MzA2NzU3OC8yOTUxMjI0OTItMDNhOWY5MWMtZjgxNy00OGMzLThiOTQtMmVjNzczYzAyNWM0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA5VDA4NDEzM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEyZWQ5NWI2ODFlZGYwM2UxZGI4NDQ1YjFjYzA0NTc5NjVhOTU4ZmYwMjBkNjYwMzE2YTQwMGFiZmViYzcxM2ImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.HuyOGPwRVaWyv3nltQmnEqbQma3zx8Avd67uKf2YbkI)

Yes, as of v2.002, those two characters follow the Unicode reference exactly, but I feel that this typeface can be better designed, which is why I am asking Adobe to adjust these two glyphs.

**Comment by Wenti-D** [comment]
> Yes, as of v2.002, those two CN glyphs follow the Unicode reference exactly, but I feel that this typeface can be better designed, which is why I am asking Adobe to adjust them.

I mean, that's true, but glyph designs usually follow the Unicode Standard. Maybe Unicode should adjust them first.🤨

**Comment by Marcus98T** [comment]
> I mean, that's true, but glyph designs usually follow the Unicode Standard. Maybe Unicode should adjust them first.🤨

That is the problem with the type foundries designing references for the Unicode standard. On the other hand, open-source typefaces like Source Han Serif can be made to look better than the Unicode reference while following glyph guidelines. I don't wish to debate this anymore.
